app/Commands/LogCommand.php

```diff
@@ -13,9 +13,11 @@ class LogCommand extends Command
 
     protected $signature = 'logs {containerId?}';
     protected $description = 'Display container logs.';
+    protected $docker;
 
-    public function handle(): void
+    public function handle(Docker $docker): void
     {
+        $this->docker = $docker;
```

Comment:
Skip assigning it as a class property, please

Reply:
Hey @mattstauffer so I left this one because it is used in the logs function in that class.